arch/riscv/core/pmp/core_pmp.c

```diff
@@ -0,0 +1,223 @@
+/*
+ * Copyright (c) 2020 BayLibre, SAS
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <kernel_internal.h>
+#include "core_pmp.h"
+#include <arch/riscv/csr.h>
+#include <stdio.h>
+
+#define PMP_SHIFT_ADDR	2
+#define PMPCFG_COUNT	4
```

Comment:
I guess that `PMP_SHIFT_ADDR` has same functionality as `[FROM|TO]_PMP_ADDR`? (because pmpaddr use addr/4 as it's register value)
If so, I think using `[FROM|TO]_PMP_ADDR` is better.
Also, does `PMPCFG_COUNT` not used in the source code?

Reply:
PMPCFG_COUNT isn't used anymore, I will remove it.
Agree to clean code by using macro only and then remove PMP_SHIFT_ADDR is it isn't used anywhere else.